src/Basque/Mews.Fiscalizations.Basque.Tests/TestFixture.cs

```diff
@@ -0,0 +1,131 @@
+                description: String1To250.CreateUnsafe("factura ejemplo TBAI"),
+                items: CreateInvoiceItems(),
+                totalAmount: 89.36m,
+                taxModes: TaxMode.GeneralTaxRegimeActivity.ToEnumerable()
+            ).Success.Get();
+        }
+
+        private static INonEmptyEnumerable<InvoiceItem> CreateInvoiceItems()
+        {
+            return NonEmptyEnumerable.Create(
+                new InvoiceItem(
+                    description: String1To250.CreateUnsafe("Artículo 1 Ejemplo"),
+                    quantity: 1,
+                    unitAmount: 23.356m,
+                    discount: 2.00m,
+                    totalAmount: 25.84m
+                ),
+                new InvoiceItem(
+                    description: String1To250.CreateUnsafe("Artículo 2 xxx"),
+                    quantity: 1.50m,
+                    unitAmount: 18.2m,
+                    discount: 0m,
+                    totalAmount: 33.03m
+                ),
+                new InvoiceItem(
+                    description: String1To250.CreateUnsafe("Artículo 3 aaaaaaaa"),
+                    quantity: 18,
+                    unitAmount: 1.40m,
+                    discount: 0m,
+                    totalAmount: 30.49m
+                )
+            );
+        }
+
+        private static InvoiceHeader CreateHeader()
+        {
+            return new InvoiceHeader(
+                number: String1To20.CreateUnsafe(RandomString(20)),
+                issued: DateTime.Now,
+                series: String1To20.CreateUnsafe(RandomString(20))
+            );
+        }
+
+        private static Subject CreateSubject()
+        {
+            return Subject.Create(Issuer, CreateReceivers(), IssuerType.IssuedByThirdParty).Success.Get();
+        }
+
+        private static INonEmptyEnumerable<Receiver> CreateReceivers()
+        {
+            return NonEmptyEnumerable.Create(new Receiver(new LocalReceiver(
+                taxpayerIdentificationNumber: TaxpayerIdentificationNumber.Create(Countries.Spain, "11111111H").Success.Get(),
+                name: Name.CreateUnsafe("AAA, BBBBB, CCCCCCCCC"),
+                postalCode: PostalCode.CreateUnsafe("01001"),
+                address: String1To250.CreateUnsafe("Eduardo Dato 1")
+            )));
+        }
+
+        private static TaxRateSummary CreateTaxRateSummary(decimal vat, decimal baseValue)
+        {
+            return new TaxRateSummary(
+                taxRatePercentage: Percentage.Create(vat).Success.Get(),
+                taxBaseAmount: Amount.Create(baseValue).Success.Get(),
+                taxAmount: Amount.Create(Math.Round(baseValue * vat / 100, 2)).Success.Get()
+            );
+        }
+
+        private static string RandomString(int length)
+        {
+            var chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+            return new string(Enumerable.Repeat(chars, length).Select(s => s[Random.Next(s.Length)]).ToArray());
```

Comment:
I'll let you read this again, so you can thing about what you want it to do 😁

Reply:
But like generally, why do we want a random string anyway? Either we can hardcode whatever we want, or we need it to be unique in which case we should use Guid.